index.js

```diff
@@ -0,0 +1,94 @@
+const url = "http://localhost:3000/players"
+const playerContainer = document.querySelector('div.player-container')
+const playerForm = document.querySelector('#new-player-form')
+
+document.addEventListener('DOMContentLoaded', () => {
+
+// Deliverable 1: Fetch Players
+   fetch(url)
+  .then(response => response.json())
+  .then(playersArray => {
+    //   console.log(playersArray)
+      allPlayer(playersArray)
+    })
+
+      function allPlayer(playersArray)  {
+      playersArray.forEach(onePlayer)}
+
+    //   allPlayer(playersArray) 
+
+    function onePlayer(player) {
+      let playerDiv = document.createElement('div')
+      playerDiv.classList.add("player")
+      playerDiv.dataset.id = player.id 
+      playerDiv.innerHTML= `
+      <h3>${player.name} (<em>${player.nickname}</em>)</h3>
+      <img src="${player.photo}" alt="${player.name}">
+      <p class="likes">${player.likes} likes</p>
+      <button class="like-button">❤️</button>
+      `
+      playerContainer.append(playerDiv)}
```

Comment:
Same note here regarding code formatting: I had to double-check to see if the playerForm event listener was written inside or outside of the onePlayer function